examples/gno.land/r/nt/boards2/v1/z_19_c_filetest.gno

```diff
@@ -0,0 +1,30 @@
+package main
+
+import (
+	"std"
+
+	boards2 "gno.land/r/nt/boards2/v1"
+)
+
+const owner = std.Address("g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5") // @test1
+
+var (
+	bid boards2.BoardID
+	tid boards2.PostID
+)
+
+func init() {
+	std.TestSetOriginCaller(owner)
+	bid = boards2.CreateBoard("test123")
+	tid = boards2.CreateThread(bid, "foo", "bar")
+}
+
+func main() {
+	println(boards2.IsThreadFrozen(bid, tid))
```

Comment:
I think OK to remove this one as threads are not created frozen, you could safely assume this value is `false`

Reply:
@jeronimoalbi I need to be sure that by default this func returns `false`.

Reply:
If so then it could be a file test for the `IsThreadFrozen` function instead, no? If there is not test already, then the new test would enforce that the assumption is met.

Reply:
@jeronimoalbi I don't want to spawn yet another filetest as we already have plenty of them and they don't support table tests and it's very hard to navigate between them.

I would prefer to put as much as possible into a single file instead.